app/categorization/work.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+import json
+from typing import Dict, Any, Union
+from langchain_community.chat_models import ChatOllama
+from langchain_core.prompts import PromptTemplate
+from langchain.chains import SequentialChain
+
+from llm_helper import (
+    is_years,
+    is_bounded,
+    is_european_decimal,
+    is_float,
+    is_integer,
+    is_iso8601,
+   
+    SexLevel,
+    AgeFormat
+
+)
+from promptTemplate import(
+    prompt,
+    Aprompt,
+    Dprompt
+)
+from llm_categorization import(
+    llm_invocation1,
+    llm_invocation2,
+    D,
+    A
+)
+def llm_invocation(result_dict: Dict[str, str]) -> Dict[str, Any]:
+
+    # Initialize model
+    llm = ChatOllama(model="gemma")
+
+    # Create prompt template
+    
+    # Create chain
+    chain = prompt |llm
+    
+    # chain = SequentialChain(
+#     chains=[chain1,chain2],
+#     input_variables=["column","content"]
+# )
+    
+
+    
+
+    age = { "pheno_age","age","Age"}
+    #key, value = list(result_dict.items())[0]
+    for key, value in result_dict.items():
+
+    # Invoke LLM
+        llm_response = chain.invoke({"column": key, "content": value,"age":age})
```

Comment:
```suggestion
        llm_response = chainGeneral.invoke({"column": key, "content": value,"age":age})
```
I did not keep track of the name for the `llm_response`